wasmbinding/test/custom_message_test.go

```diff
@@ -102,27 +103,142 @@ func (suite *CustomMessengerTestSuite) TestRegisterInterchainQuery() {
 	msg, err := json.Marshal(fullMsg)
 	suite.NoError(err)
 
-	events, data, err := messenger.DispatchMsg(ctx, contractAddress, suite.Path.EndpointA.ChannelConfig.PortID, types.CosmosMsg{
+	events, data, err := suite.messenger.DispatchMsg(suite.ctx, suite.contractAddress, suite.Path.EndpointA.ChannelConfig.PortID, types.CosmosMsg{
 		Custom: msg,
 	})
 	suite.NoError(err)
 	suite.Nil(events)
 	suite.Equal([][]byte{[]byte(`{"id":1}`)}, data)
 }
 
-func (suite *CustomMessengerTestSuite) TestSubmitTx() {
-	var (
-		neutron       = suite.GetNeutronZoneApp(suite.ChainA)
-		ctx           = suite.ChainA.GetContext()
-		contractOwner = keeper.RandomAccountAddress(suite.T()) // We don't care what this address is
-	)
+func (suite *CustomMessengerTestSuite) TestUpdateInterchainQuery() {
+	// reuse register interchain query test to get query registered
+	suite.TestRegisterInterchainQuery()
+	// Craft UpdateInterchainQuery message
+	queryID := 1
+	newUpdatePeriod := 111
+	msgStr := []byte(fmt.Sprintf(
+		`
```

Comment:
@pr0n00gler has asked me to not use raw string literals for JSON values and instead stick to `json.Marshal(<some struct>)` because of better type checking which would save us some time in a long run